ui/components/ui/form-field/form-field.js

```diff
@@ -107,6 +108,7 @@ export default function FormField({
             {error}
           </Typography>
         )}
+        {inputDetails}
```

Comment:
I notice that this (and in fact all of this) is inside of a `label`. It feels like this content doesn't serve to describe the input as a label would do, but is merely a line that sits underneath it. Can this go outside of it? If it can, do we need to modify this component — since we only have two new cases where we want to put something underneath the field, can we add an extra line by merely adding a new `div` after the FormField in BaseFeeInput and PriorityFeeInput?

Reply:
I agree let me fix this

Reply:
Fixed